summary_workflows/quantification/main.nf

```diff
@@ -3,23 +3,26 @@
 if (params.help) {
 	
 	    log.info"""
-	    ==============================================
-	    TCGA CANCER DRIVER GENES BENCHMARKING PIPELINE 
-		Author: Javier Garrayo Ventas
-		Barcelona Suercomputing Center. Spain. 2019
-	    ==============================================
+	    =====================================================
+	    APAeval QUANTIFICATION BENCHMARKING PIPELINE
+		Author(s): Yuk Kei Wan (*1,*2), Asier Gonzalez (*3)
+		*1 iRNA COSI
+		*2 Genomic Institute of Singapore, A*STAR, Singapore
+		*3 Barcelona Supercomputing Center, Barcelone, Spain
+		2021
+	    =====================================================
 	    Usage:
-	    Run the pipeline with default parameters:
+	    Run the pipeline with default parameters read from nextflow.config:
 	    nextflow run main.nf -profile docker
 	    Run with user parameters:
- 	    nextflow run main.nf -profile docker --input {driver.genes.file} --public_ref_dir {validation.reference.file} --participant_id {tool.name} --goldstandard_dir {gold.standards.dir} --cancer_types {analyzed.cancer.types} --assess_dir {benchmark.data.dir} --results_dir {output.dir}
+ 	    nextflow run main.nf -profile docker --input {execution.wf.APA.prediction.file} --public_ref_dir {validation.reference.file} --participant_id {tool.name} --goldstandard_dir {gold.standards.dir} --challenges_ids {analyzed.challenges} --assess_dir {benchmark.data.dir} --results_dir {output.dir}
 	    Mandatory arguments:
-                --input		List of cancer genes prediction
-				--community_id			Name or OEB permanent ID for the benchmarking community
-                --public_ref_dir 		Directory with list of cancer genes used to validate the predictions
+                --input		BED/TXT file with APA site information
+		--community_id			Name or OEB permanent ID for the benchmarking community
+                --public_ref_dir 		Directory with external reference file(s) for validating the structure of the output file(s) from the execution workflows (not utilized currently)
```

Comment:
This description sounds better than the ones in README and validation.py

Reply:
Yep, but this also can go now